src/App.js

```diff
@@ -15,15 +22,33 @@ function App() {
             <Route exact path="/">
               <Home />
             </Route>
+            <Route exact path="/freaks">
+              <ViewFreaks />
+            </Route>
             <Route exact path="/freaks/:id">
               <ViewFreakPage />
             </Route>
           </Switch>
         </div>
         <div className="app__nav">
-          <button type="button" className="app__agile">
-            <img className="app__agile-img" src="https://d30anih4i5atxe.cloudfront.net/uploads/bc2a1f67-7297-4ad4-ba62-d70042ad43cc.png" alt="Nav" />
-          </button>
+          <Link to="/">
+            <button type="button" className="app__agile">
+              <FontAwesomeIcon icon="home" />
+              <p>Home</p>
+            </button>
+          </Link>
+          <Link to="/freaks">
+            <button type="button" className="app__agile">
+              <img className="app__agile-img" src={ logo } alt="Nav" />
+              <p>Freaks</p>
+            </button>
+          </Link>
+          <Link to="/">
+            <button type="button" className="app__agile">
+              <FontAwesomeIcon icon="tasks" />
+              <p>Projects</p>
```

Comment:
BEM - Block, Element, Modifier, explain how "agile" is an `element` of the "app" `block`.